web/src/components/speaker/Links.vue

```diff
@@ -13,6 +13,9 @@
       <!--eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
       <v-icon>web</v-icon>
     </social-button>
+    <social-button :link="priorPresentation">
+      <i class="fab fa-youtube" />
```

Comment:
We don't want to box ourselves into youtube, as event organizers might also upload to vimeo or other platforms. Let's use the material icon [ondemand_video](https://material.io/resources/icons/?icon=ondemand_video&style=baseline).

```suggestion
       <!--eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
      <v-icon>ondemand_video</v-icon>
```